src/V3Os.cpp

```diff
@@ -43,6 +43,7 @@
 #include <memory>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/wait.h>
```

Comment:
Thanks for the PR, this actually is now included 12 lines below. I wonder why that did not get picked up (see #2353)

Reply:
Oh! That's my mistake for not checking master. Apologies - I was working off the `v4.034` tag, as I was trying to update the OpenBSD package to use that version. I can remove this commit. :-)